android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -218,4 +218,25 @@ public void setTrackUncaughtExceptions(Boolean enabled){
             tracker.enableExceptionReporting(enabled);
         }
     }
+
+
+    @ReactMethod
+    public void setAnonymizeIp(Boolean enabled){
+        Tracker tracker = getTracker(_trackingId);
+
+        if (tracker != null)
+        {
+            tracker.setAnonymizeIp(enabled);
+        }
+    }
+
+    @ReactMethod
+    public void setOptOut(Boolean enabled){
+      GoogleAnalytics analytics = getAnalyticsInstance();
+
+      if (analytics != null)
+      {
+            analytics.setAppOptOut(true);
```

Comment:
This should probably use the `enabled` parameter you send in, rather than just `true`